pkg/controllers/deprovisioning/suite_test.go

```diff
@@ -1044,9 +1044,9 @@ var _ = Describe("Delete Node", func() {
 			ObjectMeta: metav1.ObjectMeta{
 				Labels: map[string]string{
 					v1alpha5.ProvisionerNameLabelKey: prov.Name,
-					v1.LabelInstanceTypeStable:       mostExpensiveInstance.Name(),
-					v1alpha5.LabelCapacityType:       mostExpensiveOffering.CapacityType,
-					v1.LabelTopologyZone:             mostExpensiveOffering.Zone,
+					v1.LabelInstanceTypeStable:       leastExpensiveInstance.Name(),
```

Comment:
Had to change a bunch of tests as multi-node consolidation would now prefer for all of these tests with two expensive nodes, to delete both and launch a single cheaper replacement.